diff -aurN qsynth-0.0.1.orig/Makefile.in qsynth-0.0.1/Makefile.in
--- qsynth-0.0.1.orig/Makefile.in	2003-10-30 23:26:46.000000000 +0100
+++ qsynth-0.0.1/Makefile.in	2003-11-20 13:50:12.000000000 +0100
@@ -27,8 +27,8 @@
 	@make -f $(name).mak $(name)
 
 install:	$(name) icons/$(name).xpm
-	@install -vD $(name) $(prefix)/bin/$(name)
-	@install -vD icons/$(name).xpm $(prefix)/share/icons/$(name).xpm
+	@install -vD $(name) $(DESTDIR)$(prefix)/bin/$(name)
+	@install -vD icons/$(name).xpm $(DESTDIR)$(prefix)/share/pixmaps/$(name).xpm
 
 uninstall:	$(prefix)/bin/$(name)
 	@rm -vf $(prefix)/bin/$(name)
