diff -dur qsynth-0.4.0.orig/src/qsynthOptions.cpp qsynth-0.4.0/src/qsynthOptions.cpp
--- qsynth-0.4.0.orig/src/qsynthOptions.cpp	2015-09-07 16:59:23.000000000 +0200
+++ qsynth-0.4.0/src/qsynthOptions.cpp	2015-12-11 18:01:50.031289071 +0100
@@ -80,7 +80,7 @@
 
 	// Load defaults...
 	m_settings.beginGroup("/Defaults");
-	sSoundFontDir  = m_settings.value("/SoundFontDir").toString();
+	sSoundFontDir  = m_settings.value("/SoundFontDir", QString("/usr/share/soundfonts")).toString();
 	bPresetPreview = m_settings.value("/PresetPreview", false).toBool();
 	m_settings.endGroup();
 
